src/cow_utils.c

```diff
@@ -91,7 +91,9 @@ int setfile(const char *path, struct stat *fs)
 	* chown.  If chown fails, lose setuid/setgid bits.
 	*/
 	if (chown(path, fs->st_uid, fs->st_gid)) {
-		if (errno != EPERM) {
+		/* EPERM if no permissions
+		 * EINVAL if user was nobody or group was nogroup */
+		if (errno != EPERM && errno != EINVAL) {
```

Comment:
`chown` throws `EINVAL` if the uid represents the user `nobody`. This user is commonly used for marking that we don't know which user it is on an NFS share or a user namespace. Previously the whole cow-copy operation failed here if the source file was owned by nobody.